Makefile

```diff
@@ -38,18 +38,24 @@ scripts/terraform-outputs-gce.sh: check-env-vars gce/${DEPLOY_ENV}.tfstate
 prepare-provision-aws: set-aws manifests/templates/outputs/terraform-outputs-aws.yml scripts/terraform-outputs-aws.sh prepare-provision
 prepare-provision-gce: set-gce manifests/templates/outputs/terraform-outputs-gce.yml scripts/terraform-outputs-gce.sh prepare-provision
 prepare-provision: check-env-vars bastion
+	scp -r -oStrictHostKeyChecking=no cf-manifest/deployments \
+	    ubuntu@${bastion}:
 	scp -r -oStrictHostKeyChecking=no manifests/templates \
 	    manifests/generate_bosh_manifest.sh \
 	    ubuntu@${bastion}:
 	scp -r -oStrictHostKeyChecking=no scripts ubuntu@${bastion}:
+	PASSWORD_STORE_DIR=~/.paas-pass pass ${ROOT_PASS_DIR}/cloudfoundry/cf-secrets.yml | \
+	    ssh -oStrictHostKeyChecking=no ubuntu@${bastion} 'cat > templates/cf-secrets.yml'
```

Comment:
`cf-secrets.yml` in the master branch of the paas-pass repo contains a spiff merge operator which trips up spruce. Because we can't update this in the paas-pass repo without breaking trial deployments perhaps we could use `sed` here to remove that one line. and unblock this deployment. I realise that's a hacky approach, but this isn't going to be long-lived, so I think it's ok to do here.